pkgs/utils/test_utils/test_utls.go

```diff
@@ -85,16 +66,13 @@ func CreateTestOperator(t *testing.T, id uint64, version string) *TestOperator {
 	priv, err := rsaencryption.ConvertPemToPrivateKey(string(pv))
 	require.NoError(t, err)
 	r := chi.NewRouter()
-	db, err := kv.NewInMemory(logging.TestLogger(t), basedb.Options{
-		Reporting: true,
-		Ctx:       context.Background(),
-		Path:      t.TempDir(),
-	})
 	require.NoError(t, err)
 	operatorPubKey := priv.Public().(*rsa.PublicKey)
 	pkBytes, err := crypto.EncodePublicKey(operatorPubKey)
-	require.NoError(t, err)
-	swtch := operator.NewSwitch(priv, logger, db, []byte(version), pkBytes, id)
+	if err != nil {
+		panic(err)
```

Comment:
more suitable to do this:
```suggestion
	require.NoError(t, err)
```